web/public/locales/en.json

```diff
@@ -1508,6 +1508,10 @@
       "countryName": "USA",
       "zoneName": "Alaska"
     },
+    "US-AK-SAP": {
```

Comment:
Not exactly sure what the policy for zone names is... It might make sense to use `SEAPA` instead of `SAP`, as that's the name of the power agency. @VIKTORVAV99, what do you think?

Reply:
SEAPA would be slightly easier to remember and work with.